components/table/Sidebar.jsx

```diff
@@ -43,6 +45,21 @@ const SidebarHeader = ({ children, ...passProps }) => {
   )
 }
 
+const SidebarBody = ({ children, ...passProps }) => (
```

Comment:
This seems to be useless. I would prefer to avoid this and just inline children. Minor, I will probably fix it myself later.

Reply:
it is there just because it looks cool. 😄 (Not a good reason)